modules/cnv-example-cr-hostpath-provisioner.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-configuring-local-storage-for-vms.adoc
+
+[id="cnv-example-cr-hostpath-provisioner_{context}"]
+= Example: Custom resource file for the hostpath provisioner
+
+The following YAML file is used to create a custom resource for the hostpath
+provisioner. It includes sample values; ensure that you replace them with your
+own values where applicable. +
+[NOTE]
+====
+All values shown in this example are required.
+====
+
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1alpha1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+spec:
+  imagePullPolicy: IfNotPresent
+  pathConfig:
+    path: "</path/to/backing/directory>" <1>
+    useNamingPrefix: "false" <2>
+----
+<1> Specify the backing directory where you want the provisioner to create PersistentVolumes (PV).
```

Comment:
I think if you add these to the example in the procedure itself we don't need the example at all. It seems to be the only difference.